Dockerfile-dev

```
@@ -3,15 +3,19 @@
 
 FROM openjdk:8-jdk
 ENV JENKINS_UC https://updates.jenkins.io
+ENV CASC_JENKINS_CONFIG file:///casc/jenkins.yaml
```

It would be nice to point this to a directory, so that the configuration can be split into multiple files?

---

I tried that, but CasC read the directory as a space-separated list of the file names, which is invalid YAML. I'll take another look if there's a different way to point it to a directory.

---

I agree that it should be a directory by default. If it does not work, it might be a bug in JCasC.

```suggestion
ENV CASC_JENKINS_CONFIG /casc
```

---

The jenkins helm chart relies on the directory behaviour: https://github.com/helm/charts/tree/master/stable/jenkins

---

Updating the JCasC plugin from 0.30 to 0.32 solved the issue. This change is in the bom as well, but not yet released, so we'll need to override it until a new version is released (changed here: https://github.com/jenkinsci/bom/commit/64c6b7dd6c51d096833fcd01f04fc2551fa20f11)

---

Overriding BOM is fine for now